<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player</class>
 <widget class="QDialog" name="player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #312b1b;</string>
  </property>
  <widget class="QLabel" name="image_profile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-width: 5px;
border-color: #e68a00;
outline-style: hidden;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="connected_as_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Brush Script MT', cursive;
letter-spacing: 1px;
color: white;</string>
   </property>
   <property name="text">
    <string>Connected as:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Brush Script MT', cursive;
letter-spacing: 1px;
color: white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="options">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
   border-style: hidden;
	color: white;
	padding: 1px 0px 1px 3px;
}

QComboBox::drop-down 
{
    border: 0px;
	color: white;
}

QComboBox::down-arrow {
    image: url(:/img/img/arrow.png);
	width: 20px;
	height: 14px;
}

QComboBox QAbstractItemView {
    border-style: hidden;
    color: white;
}</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>961</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane {
	border: 2px solid #e68a00;
}

QTabBar::tab {
	background-color: transparent;
	width: 80px;
	height:80px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
	background-color: #e68a00;
}

QTabBar::tab:first {
	 image: url(:/img/img/disc.png);
	padding: 5px;
}

QTabBar::tab:last {
	 image: url(:/img/img/play-button.png);
	 padding: 5px;
}
</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QPushButton" name="add_song_button">
     <property name="geometry">
      <rect>
       <x>754</x>
       <y>300</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
image: url(&quot;:/img/img/add.png&quot;);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>721</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="songs_layout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string/>
    </attribute>
   </widget>
  </widget>
  <widget class="QSlider" name="timeElapseSlider">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
    border: 1px solid;
    height: 10px;
    margin: 0px;
    }
QSlider::handle:horizontal {
    background-color: #e68a00;
    height: 40px;
    width: 20px;
    margin: -5px 0px;
	border-radius: 10px;
    }

QSlider::add-page:horizontal {
    background: grey;
}

QSlider::sub-page:horizontal {
    background: red;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="current_song">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font-size:12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>30</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
    border: 1px solid;
    height: 10px;
    margin: 0px;
    }
QSlider::handle:horizontal {
    background-color: #e68a00;
    height: 40px;
    width: 20px;
    margin: -5px 0px;
	border-radius: 10px;
    }

QSlider::add-page:horizontal {
    background: grey;
}

QSlider::sub-page:horizontal {
    background: red;
}</string>
   </property>
   <property name="sliderPosition">
    <number>60</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>766</x>
     <y>24</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/img/volume.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="play">
   <property name="geometry">
    <rect>
     <x>534</x>
     <y>102</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/img/play.png);
background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pause">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>102</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/img/pause.png);
background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="song_photo">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/img/sound.png);
width: 81;
height: 71;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
